src/pages/accountsettings/AccountSettings.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, Component } from "react";
```

Comment:
I assume that this component is not finished and that you will complete it as soon as you have the Parse ready, in order to retrieve the person info and edit them.

Reply:
yes